src/game/scenes/mechlab/lab_menu_main.c

```diff
@@ -144,7 +144,8 @@ void lab_menu_focus_arena(component *c, bool focused, void *userdata) {
         sd_chr_enemy *enemy = mechlab_next_opponent(s);
         if(enemy) {
             char tmp[100];
-            snprintf(tmp, 100, lang_get(537), enemy->pilot.name);
+            // XXX lang string provided format string
```

Comment:
What's the XXX for? Are there safety problems?

Reply:
Yes, and I'd like users to be free to install untrusted game data from the internet with less risk.
Untrusted format strings can do terrible things (stack corruption -> rop -> ace), and are inflexible in the order of arguments.
Later (not in this PR), i want all %s's to be replaced with newsroom-style substitutions.